cmd/lvh/main.go

```diff
@@ -36,6 +36,7 @@ func init() {
 			},
 		},
 	)
+	rootCmd.SetOut(os.Stdout)
```

Comment:
ah no! It's just that by default, all `cmd.Print` in cobra goes to stderr instead of stdout, it's a legacy thing https://github.com/spf13/cobra/issues/1490. It's just so that `fmt.Print` is equivalent to `cmd.Print`. But I like using `cmd.Print` because you can easily redirect stuff in tests.

Reply:
I tried and it seems `lvh --help` is always sent to stdout, regardless of this fix, but it's a bit more complicated than I thought https://github.com/spf13/cobra/issues/1708.

To be honest, it was just so that I can use `cmd.Print` and send to stdout. If we don't want this, I can just use `fmt.Print` since there are no tests here anyway.

Reply:
:+1: TIL `cmd.Print()`. I always just use `fmt.Fprintf(os.Stdout, ...)` so I know for sure where it's going :sweat_smile: . Now that I look more closely, nothing about usage changes, so I'm not sure why the cobra API description says that. I withdraw my feedback here.